ping-protect/src/main/java/org/forgerock/android/auth/PingOneProtectEvaluationCallback.kt

```diff
@@ -44,6 +42,7 @@ open class PingOneProtectEvaluationCallback : AbstractCallback {
      */
     fun setSignals(value: String) {
         super.setValue(value, 0)
```

Comment:
Any side effect for setting this for marketplace node? There should be no input for marketplace node.

Reply:
good catch!